nxcomp/Log.cpp

```diff
@@ -0,0 +1,81 @@
+
+#include <stdio.h>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+
+#include "Log.h"
+
+NXLog nx_log;
+
+
+NXLog::NXLog(char *filename)
+{
+	std::fstream *file = new std::fstream();
```

Comment:
Any reason for using fstream instead of ofstream (which was previously used). We do not need to operate on files in read mode, do we?